index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/4dc3d7e55b.js" crossorigin="anonymous"></script>
+    <title>Document</title>
+</head>
+<body>
+    <header class="Header">Josué</header>
+    <button class="menu">
+    <i class="fa-solid fa-bars"></i></button>
+    <section class="section">
+        <h1>I´m Josué Glad to see you!</h1>
+        
+    </section>
```

Comment:
- [ ] Section tag should be wrapping all the content from the `h1` you currently has, until the social media icons, as a whole block of code